packages/components/src/typings.d.ts

```diff
@@ -1,16 +1,13 @@
+import { Content, MessageType } from '@botpress/messaging-server/content-types'
 import { AxiosInstance } from 'axios'
 import { InjectedIntl } from 'react-intl'
-import { Settings as CarouselSettings } from 'react-slick'
```

Comment:
I don't think this package should depend on the server code! You can just copy paste the file from the server for now, and we'll probably move it to the `base` package later.

Reply:
done